### ISSUE TITLE
Note Blocks (alarm)

### ISSUE THREAD

**Issue by BedoAkgun** [Anuken/Mindustry-Suggestions#1108]
i was playing v6 survival with other people but energy just go negative. We use lancers and meltdowns so liches destroyed the defence and i want a sound block that will give sound when a processor says. So it will be better and people can know whats happening and they will do the things to fix the things.

**Comment by itcannotbe** [comment]
1. RIP Template
2. Duplicate of #868 
3. It's on the trello board.

**Comment by itcannotbe** [comment]
All in all this summarizes this suggestion.
![Screenshot_20201108_224833](https://user-images.githubusercontent.com/45280919/98468242-93d78f00-2214-11eb-81da-0f87e9ed1d14.jpg)

**Comment by BedoAkgun** [comment]
...

**Comment by BedoAkgun** [comment]
oh i didnt saw #868 but there is a poorly made art of this block

**Comment by BedoAkgun** [comment]
![Yeni-Bit-Eşlem-Resmi](https://user-images.githubusercontent.com/65866977/98468524-e81a4900-21eb-11eb-98c7-1e95b8cc17a3.jpg)

**Comment by BedoAkgun** [comment]
i only looked the posts after #900

**Comment by Catchears** [comment]
just use search

**Comment by DeltaNedas** [comment]
why didnt you use the template

**Comment by TheSlaylord** [comment]
Also duplicate of #810
